src/pages/library/DataBrowserDetails.js

```diff
@@ -223,10 +261,81 @@ export const SidebarComponent = ({ dataObj, id }) => {
     showRequestAccessModal && h(RequestDatasetAccessModal, {
       datasets: [dataObj],
       onDismiss: () => setShowRequestAccessModal(false)
+    }),
+    datasetNotSupportedForExport === true && h(Modal, {
```

Comment:
Nit: It looks to me like `datasetNotSupportedForExport` is always `true` or `false`, so I don't think the `=== true` is necessary. Did you find something different in your testing?